Libraries/CustomComponents/Navigator/NavigatorNavigationBar.js

```diff
@@ -74,6 +74,11 @@ class NavigatorNavigationBar extends React.Component {
     navigationStyles: NavigatorNavigationBarStyles,
   };
 
+  constructor(props){
+    super(props);
+    this._reset = this._reset.bind(this)
```

Comment:
Arrow functions used as class properties are bound to `this` by default, so I don't think this should make any difference. Have you tested your app with/without this change and seen if it affects the behavior?